tests/test_views.py

```diff
@@ -0,0 +1,91 @@
+from datetime import datetime
+
+import pytest
+from flask import url_for
+
+from devlog.utils.text import slugify
+
+
+def test_index_no_posts(client):
+    url = url_for('main.index')
+    rv = client.get(url)
+    assert rv.status_code == 200
+    assert 'żadnych postów' in rv.text
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/jarek.zgoda/devlog/pullRequest?prid=5774733)